docs/robot_advanced.rst

```diff
@@ -117,32 +117,68 @@ With the ``PageObjects`` library, you can define classes that represent page obj
 
-* ``self.object_name``: The name of the object related to the class. This is defined via the ``object_name`` parameter to the ``pageobject`` decorator. Do not add the namespace prefix in the decorator. This attribute will automatically add the prefix from the ``cumulusci.yml`` file when necessary.
-* ``self.builtin``: A reference to the Robot Framework ``BuiltIn`` library that can be used to directly call built-in keywords. Any built-in keyword can be called by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.builtin.log`` and ``self.builtin.get_variable_value``).
-* ``self.cumulusci``: A reference to the CumulusCI keyword library. Call any keyword in this library by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.cumulusci.get_org_info``).
-* ``self.salesforce``: A reference to the Salesforce keyword library. Call any keyword in this library by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.salesforce.wait_until_loading_is_complete``).
-* ``self.selenium``: A reference to SeleniumLibrary. Call any keyword in this library by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.selenim.wait_until_page_contains_element``).
+The following example illustrates how to create a ``Listing`` page object for ``CustomObject__c``.
+
+.. code-block:: python
+
+    from cumulusci.robotframework.pageobjects import ListingPage, pageobject
 
+    @pageobject(page_type='Listing', object_name='CustomObject__c')
+    class CustomObjectListingPage(ListingPage):
+        ...
+
+Using object aliases
+""""""""""""""""""""
+
+Within a test, if you want to refer to the page object with a more human-readable name such as ``Custom Object`` rather than ``CustomObject__c`` you can do so by setting ``object_name`` to ``Custom Object`` and then defining ``_object_name`` in the class, like in the following example.
+
+.. code-block:: python
+
+    from cumulusci.robotframework.pageobjects import ListingPage, pageobject
+
+    @pageobject(page_type = 'Listing', object_name = 'My Object')
+    class CustomObjectListingPage(ListingPage):
+        _object_name = 'MyObject__c'
+        ...
+
+By using an alias, you can reference the page object with either the alias or the actual object name. For example, the following two uses of ``Go to page`` are identical:
+
+.. code-block:: robot
+
+    Go to page  Listing  My Object
+    Go to page  Listing  MyObject__c
 
 
 Page Object Base Classes
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-Presently, CumulusCI provides the following base classes, which should be used for all classes that use the ``pageobject`` decorator:
+Presently, CumulusCI provides the following base classes, which should be used for all classes that use the ``pageobject`` decorator. These base classes may be imported from ``cumulusci.robotframework.pageobjects``.
```

Comment:
```suggestion
CumulusCI provides the following base classes, which should be used for all classes that use the ``pageobject`` decorator. You can import these base classes from ``cumulusci.robotframework.pageobjects``.
```
If you don't like "You can import," I'm okay with "These base classes can be imported."